procsim/biomass/level1_product_generator.py

```diff
@@ -81,53 +82,53 @@ def get_params(self) -> Tuple[List[tuple], List[tuple], List[tuple]]:
     def _generate_product(self):
         name_gen = self._create_name_generator(self._hdr)
         dir_name = name_gen.generate_path_name()
-        self._hdr.set_product_filename(dir_name)
+        self._hdr.initialize_product_list(dir_name)
         self._logger.info('Create {}'.format(dir_name))
 
-        # List with files: array of directory, suffix, extension
-        files = [
-            (['annotation'], 'annot', 'xml'),
-            (['annotation', 'calibration'], 'cal', 'xml'),
-            (['annotation', 'calibration'], 'ant', 'dat'),
-            (['annotation', 'calibration'], 'noise', 'dat'),
-            (['annotation', 'navigation'], 'orb', 'xml'),
-            (['annotation', 'navigation'], 'att', 'xml'),
-            (['annotation', 'geometry'], 'geoloc', 'tiff'),
-
-            (['preview'], 'map', 'kml'),
-            (['preview'], 'ql', 'png'),
-
-            (['measurement', 'rfi'], 'rfi', 'tbd'),
-            (['measurement', 'ionosphere'], 'iono', 'tiff'),
-
-            (['schema'], 'Annotation', 'xsd'),
-            (['schema'], 'Orbit', 'xsd'),
-            (['schema'], 'Attitude', 'xsd')
+        annot_schema = GeneratedFile(['schema'], 'Annotation', 'xsd')
+        orb_schema = GeneratedFile(['schema'], 'Orbit', 'xsd')
+        att_schema = GeneratedFile(['schema'], 'Attitude', 'xsd')
+
+        bin_files = [
```

Comment:
Not really bin files.

Reply:
Renamed to be simply `files` in 7f02b4e12aa566ae820eacd3091d811660851272. I had previously made the distinction between bin files and representation files, but the latter do not have their own list anymore.